### PR TITLE
feat(route): add 腾讯研究院

### LINKED ISSUE #9141: 腾讯研究院
### 类型

新媒体

### 网站地址

https://www.tisi.org/

### 网站描述

是腾讯公司设立的社会科学研究机构。非盈利性学术网站，所有文章约为学术研究用途。

### 需要生成什么内容？

首页文章

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #9141

## 完整路由地址 / Example for the proposed route(s)

```routes
/tisi/latest
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [X] 新的路由 New Route
  - [X] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [X] 文档说明 Documentation
  - [X] 中文文档 CN
  - [ ] 英文文档 EN
- [X] 全文获取 fulltext
  - [X] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [X] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [X] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/tisi/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+                return item;
+            })
+        )
+    );
+
+    ctx.state.data = {
+        title: '腾讯研究院 - 最近更新',
+        link: url,
+        item: await Promise.all(items),
```
Comment (TonyRL) [inline lib/v2/tisi/index.js:36]:
```suggestion
        item: items,
```

File: lib/v2/tisi/radar.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    'tisi.org/': {
```
Comment (TonyRL) [inline lib/v2/tisi/radar.js:2]:
```suggestion
    'tisi.org': {
```

File: lib/v2/tisi/radar.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        _name: '腾讯研究院',
+        '.': [
+            {
+                title: '最近更新',
+                docs: 'https://docs.rsshub.app/new-media.html#topys-guan-jian-zi',
```
Comment (TonyRL) [inline lib/v2/tisi/radar.js:7]: Wrong docs link

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/tisi/radar.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ module.exports = {
         '.': [
             {
                 title: '最近更新',
-                docs: 'https://docs.rsshub.app/new-media.html#topys-guan-jian-zi',
+                docs: 'https://docs.rsshub.app/new-media.html#teng-xun-yan-jiu-yuan-zui-jin-geng-xin',
```
Comment (TonyRL) [inline lib/v2/tisi/radar.js:7]: This should point to the H2 header, not the H3 header

Comment (Fatpandac) [reply lib/v2/tisi/radar.js:7]: I think H3 link is clearer than H2 link.

Comment (TonyRL) [reply lib/v2/tisi/radar.js:7]: https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze-bian-xie-gui-ze-docs

Comment (Fatpandac) [reply lib/v2/tisi/radar.js:7]: OK

**Pull request merged**